src/transformers/models/ernie_m/configuration_ernie_m.py

```diff
@@ -0,0 +1,163 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" ErnieM model configuration"""
+# Copied from original paddlenlp repository.(https://github.com/PaddlePaddle/PaddleNLP/blob/develop/paddlenlp/transformers/ernie_m/configuration.py)
+
+from __future__ import annotations
+
+from typing import Dict
+
+from ...configuration_utils import PretrainedConfig
+
+
+__all__ = ["ERNIE_M_PRETRAINED_INIT_CONFIGURATION", "ErnieMConfig", "ERNIE_M_PRETRAINED_RESOURCE_FILES_MAP"]
+
+
+ERNIE_M_PRETRAINED_INIT_CONFIGURATION = {
+    "ernie-m-base_pytorch": {
+        "attention_probs_dropout_prob": 0.1,
+        "hidden_act": "gelu",
+        "hidden_dropout_prob": 0.1,
+        "hidden_size": 768,
+        "initializer_range": 0.02,
+        "max_position_embeddings": 514,
+        "num_attention_heads": 12,
+        "num_hidden_layers": 12,
+        "vocab_size": 250002,
+        "pad_token_id": 1,
+        "layer_norm_eps": 1e-05,
+    },
+    "ernie-m-large_pytorch": {
+        "attention_probs_dropout_prob": 0.1,
+        "hidden_act": "gelu",
+        "hidden_dropout_prob": 0.1,
+        "hidden_size": 1024,
+        "initializer_range": 0.02,
+        "max_position_embeddings": 514,
+        "num_attention_heads": 16,
+        "num_hidden_layers": 24,
+        "vocab_size": 250002,
+        "pad_token_id": 1,
+        "layer_norm_eps": 1e-05,
+    },
+}
+
+ERNIE_M_PRETRAINED_RESOURCE_FILES_MAP = {
+    "model_state": {
+        "ernie-m-base_pytorch": "https://huggingface.co/susnato/ernie-m-base_pytorch/blob/main/pytorch_model.bin",
+        "ernie-m-large_pytorch": "https://huggingface.co/susnato/ernie-m-large_pytorch/blob/main/pytorch_model.bin",
+    }
+}
+ERNIE_M_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "susnato/ernie-m-base_pytorch": "https://huggingface.co/susnato/ernie-m-base_pytorch/blob/main/config.json",
+    "susnato/ernie-m-large_pytorch": "https://huggingface.co/susnato/ernie-m-large_pytorch/blob/main/config.json",
+}
+
+
+class ErnieMConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [*ErnieModel*]. It is used to instantiate a ERNIE
+    model according to the specified arguments, defining the model architecture. Instantiating a configuration with the
+    defaults will yield a similar configuration to that of the ERNIE ernie-3.0-medium-zh architecture. Configuration
+    objects inherit from [*PretrainedConfig*] and can be used to control the model outputs. Read the documentation from
```

Comment:
```suggestion
    defaults will yield a similar configuration to that of the ERNIE ernie-3.0-medium-zh architecture. Configuration
    objects inherit from [*PretrainedConfig*] and can be used to control the model outputs. Read the documentation from
```
to adapt to specify that it is Ernie-M